qa/rpc-tests/zmq_test.py

```diff
@@ -45,13 +46,15 @@ def run_test(self):
             msg = self.zmqSubSocket.recv_multipart()
             topic = msg[0]
             body = msg[1]
-
+            print("tx message:\n",topic, "\n", bytes_to_hex_str(body))
+            assert_equal(topic, b"hashtx")
             msg = self.zmqSubSocket.recv_multipart()
             topic = msg[0]
             body = msg[1]
             blkhash = bytes_to_hex_str(body)
-
+            assert_equal(topic, b"hashblock")
             assert_equal(genhashes[0], blkhash) #blockhash from generate must be equal to the hash received over zmq
+            print("block message\n",topic, "\n", blkhash)
```

Comment:
ditto.

Reply:
I deliberately left these in.  They are pretty useful and just 2 lines (not in a loop).  WRT using the logging machinery, this is in qa test code.  Typically these tests just print to log.  I added the few exceptions myself for some tests that log a LOT of info.